src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -8094,6 +8094,24 @@ void N(string? s)
             NullabilityAnalysis(string.Format(FeaturesResources._0_is_not_null_here, "s")));
     }
 
+    [Fact]
+    public async Task NullableParameterThatIsOblivious()
+    {
+        await TestWithOptionsAsync(TestOptions.Regular8,
+            """
+            class X
+            {
+                void N(string s)
+                {
+            #nullable enable
+                    string s2 = $$s;
```

Comment:
While maybe obvious, it feels like it would be interesting to demonstrate a similar scenario checking a usage of `s2` on the next line. I would expect it to show "s2 is not null here", which is fine--that shows we have moved the cliff out from where it used to be.